.travis.yml

```diff
@@ -32,7 +26,10 @@ after_failure:
 env:
   global:
     - SEED=1
-    - secure: "barHldniAfXyoWOD/vcO+E6/Xm4fmcaUoC9BeKW+LwsHqlDMLvugaJnmLXkSpkbYhVL61Hzf3bo0KPJn88AFc5Rkf8oYHPjH4adMnVXkf3B9ghHCgznqHsAH3choo6tnPxaFgOwOYmLGb382nQxfE5lUdvnM/W/psQjWt66A1+k="
+    - secure: "FrI5d2s+ckckC17T66c8jm2jV6i2DkBPU5nyWzwbedjmEBeocREfQLd/x8yKpPzLDz7ghOvr+/GQvsPPn0dVkGlNzm3Q+hGHc/ujnASuUtGrcuMM+0ALnJ3k4rFr9xEvjJeWb4SmhJO5UCAZYvTItW4k7+bj9L+R6lt3TzQbXzg="
+
+before_install:
```

Comment:
Is this `before_install` step necessary?

Reply:
This step is been mentioned in https://scan.coverity.com/projects/armmbed-mbedtls/builds/new?tab=travis_ci

Reply:
But is the step required? Maybe we have their cert already in the Travis machines.

Reply:
I'm unsure about it, have to perform trial and error again to see if we can get away with it... In the past, trials were made with this. I can't confirm the certificate is already present in Travis, probably due to my account limitations.

Reply:
Let's remove the `before_install` step and do a test run.